strr-platform-web/app/stores/platformApplication.ts

```diff
@@ -1,31 +1,59 @@
 export const useStrrPlatformApplication = defineStore('strr/platformApplication', () => {
+  const { $strrApi } = useNuxtApp()
```

Comment:
I always find this nice.

Reply:
yeah I really like this way. Had not seen this before working with the connect layer